src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -377,13 +371,10 @@ public static void TCP_ServiceCertFailedCustomValidate_Throw_Exception()
         Assert.Equal(MyCertificateValidator.exceptionMsg, exception.Message);
     }
 
-#if FULLXUNIT_NOTSUPPORTED
     [Fact]
```

Comment:
Reviewers: the [Fact] was inside the #if, so this test wasn't running except in NET Native.  Fixed now.